.github/workflows/lint-test.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       fail-fast: false
       matrix:
         os: ["ubuntu-latest", "macos-latest"]
-        python-version: ["3.6", "3.7", "3.8", "3.9"]
+        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10"]
```

Comment:
I think conda dropped support for  Python 3.6, yes?

Reply:
Conda-forge is no longer building new packages Python 3.6, so if any of the conda dependencies come from there you may have problems with missing versions

https://conda-forge.org/docs/user/announcements.html#announcements

> 2021-10-04

>    python 3.6 is now dropped when building conda-forge packages

>    python 3.6 is end-of-life in December 2021 and we are dropping support for it early to avoid having to rebuild packages as part of python 3.10 migration as that would save lots of CI resources.